project_precommit_check

```diff
@@ -177,7 +177,13 @@ def is_correct_swift_version(swiftc, compatibility_version):
 
 
 def check(project, swift_branch, swiftc, compatibility_version):
-    if platform.system() == 'Darwin':
+    # Please, rewrite this script.
+    # Writing into function parameters is a bad manner. You can do better.
+    # Please, rewrite it.
+    # Thanks!
```

Comment:
See #365.

Reply:
@clackary 
I would like to see a message before compatibility version check.

```
    swiftc_path = os.path.abspath(swiftc)
    common.debug_print("checking swiftc_path -> " + swiftc_patch + " and " + comparability_version + " <- compatibility version")
    if not is_correct_swift_version(swiftc_path, compatibility_version):
```

Maybe it is better to add this output to `is_correct_swift_version()`

Reply:
The full path to swifc is already printed in `is_correct_swift_version()`.

https://github.com/apple/swift-source-compat-suite/blob/273551372c4aa5c406b7833991ae77ee39d11bcc/project_precommit_check#L160